### ISSUE TITLE
Is that a overfitting problem?

### ISSUE THREAD

**Issue by YycYoung** [ultralytics/ultralytics#1539]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I am using yolov8s pre-trained model to train the Montreal-traffic-images dataset, but the results hardly change much by hyperparameter tuning. As shown in the figure, does this mean overfitting? How else should I tune the hyperparameters? Or is it enough to train to this level?
![results](https://user-images.githubusercontent.com/85680484/226605154-92b54ab3-625d-42da-aa18-26241d07ad3c.png)

**Comment by YycYoung** [comment]
> No, your val results are not increasing, so you are not overfitting. Train 2x longer.

Thank you for your reply！

When the val loss and mAP curves remain stable, but the train loss continues to decrease, does this also mean that the model has been trained？

**Comment by glenn-jocher** [comment]
@YycYoung yeah mostly. If you really want to fully train though you want your val losses to start to increase. Then you know you've trained enough and longer training won't help.

**Comment by YycYoung** [comment]
> @YycYoung yeah mostly. If you really want to fully train though you want your val losses to start to increase. Then you know you've trained enough and longer training won't help.

I got it! Thanks for your reply!
